draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -295,6 +296,38 @@ For example:
 Content-Dictionary: :pZGm1Av0IEBKARczz7exkNYsZb8LzaMrV7J32a2fFG4=:
 ~~~
 
+# The 'compression-dictionary' Link Relation Type
+
+This specification defines the 'compression-dictionary' link relation type
+{{WEB-LINKING}} that provides a mechanism for a HTTP response to provide a URL
+for a compression dictionary that is related to, but not directly used by the
+current HTTP response.
+
+The 'compression-dictionary' link relation type indicates that fetching and
+caching the specified resource is likely to be beneficial as future requests
+are likely to support using the provided compression dictionary for encoding
+responses.
+
+Clients SHOULD fetch the provided resource at a time that they determine would
+be appropriate.
+
+The response to the fetch for the compression dictionary is responsible for
+including the appropriate "Use-As-Dictionary" and caching response headers. The
+link relation only provides the mechanism for triggering the fetch of the
+dictionary.
+
+## Example
+For example:
+
+~~~ http-message
+Link: <https://example.org/dictionary.dat>;
+      rel="compression-dictionary"
```

Comment:
```suggestion
Link: <https://example.org/dictionary.dat>; rel="compression-dictionary"
```

I don't think that this blows the line limit.  If it does, shorten the URL before wrapping.